tests/README.md

```diff
@@ -570,6 +570,29 @@ ipdb> test_microvm.help.gdbserver()
 
 You get some instructions on how to run GDB to attach to gdbserver.
 
+## How to run tests with a different version of Firecracker
+
+The integration tests usually compile Firecracker as part of the test
+initialization. But there's an option in case we want to run the tests against a
+different version of Firecracker, for example a previous release:
+
+```sh
+./tools/devtool test -- --binary-dir ../v1.4.1
+```
+
+## How to run tests outside of Docker
+
+Tested in Ubuntu 22.04, AL2023. AL2 does not work due to an old Python (3.8).
+
+```sh
+sudo yum install python3-pip
+sudo pip3 install pytest ipython requests psutil tenacity filelock "urllib3<2.0" requests_unixsocket aws_embedded_metrics pytest-json-report pytest-timeout
```

Comment:
Probably needs a `--break-system-packages` for newer python versions. Otherwise, maybe we can use poetry here? And point people towards using a venv instead of installing packages globally? Messing up the system python is fine for us because we can just recycle our EC2 dev instances, but I wouldn't want someone to mess up their local python installation on accident, haha.

Reply:
Yes in Ubuntu24.04 I think, but IIRC worked fine in AL2023 and Ubuntu22.04